var/spack/repos/builtin/packages/slepc/package.py

```diff
@@ -156,19 +158,20 @@ def run_hello_test(self):
         """Run stand alone test: hello"""
         test_dir = self.test_suite.current_test_data_dir
 
-        if not os.path.exists(test_dir):
-            print('Skipping slepc test')
+        if not os.path.isfile(join_path(test_dir, 'hello.c')):
+            tty.msg('Skipping slepc test: failed to find hello.c')
```

Comment:
If this is supposed to be there perhaps a tty.die is more appropriate to raise an error?

Reply:
We generally don't want stand-alone tests to kill execution.  Unfortunately, I can't recall, off-hand, if using `tty.die` in this context translates into an exception that is caught.

Reply:
I do think it would exit - so maybe that's not appropriate here. Is there a tty.warning or similar? I guess given that this doesn't exist it just won't run tests?

Reply:
There is `tty.warn`.  The `return` at line 163 will skip the tests.

We really want a better way to identify and track skipped tests.  (It would be better to have a standard convention for CDash parsing and reporting.)  There are a couple of ideas (and at least one PR) but free cycles has been an issue.